### DISCUSSION TITLE
v3版本api文档

### DISCUSSION THREAD

**Discussion by 12189108** [AlistGo/alist#2074] [Q&A]
有无好心人整理了v3版本的API文档

**Comment by anwen-anyi** [comment]
https://console-docs.apipost.cn/preview/516fda102b501212/eaa3b8943426ceb5

**Comment by anwen-anyi** [comment]
> 有无好心人整理了v3版本的API文档

大佬有把API保存下来吗

**Reply by 12189108** [reply]
没有，之前有些API混乱，不符合V3版本的，而且不全，就没保存

**Reply by anwen-anyi** [reply]
好的吧QAQ
